clang-tools-extra/clangd/tool/ClangdMain.cpp

```diff
@@ -500,6 +500,16 @@ opt<bool> EnableConfig{
     init(true),
 };
 
+opt<bool> StrongWorkspaceMode{
+    "strong-workspace-mode",
+    cat(Features),
+    desc("An alternate mode of operation for clangd, operating more closely to "
```

Comment:
Suggested reword: "operating more closely to the workspace" --> "where the clangd instance is used to edit a single workspace".